### PR TITLE
Add getAspectRatio() function to VideoMode

### PR DESCRIPTION
**Pull request by dustdfg**
## Description

Add getAspectRatio() function to VideoMode

## Tasks

* [ ] Tested on Linux
* [ ] Tested on Windows
* [ ] Tested on macOS
* [ ] Tested on iOS
* [ ] Tested on Android
* [ ] Add detailed description
* [ ] Add some tests

## How to test this PR?

Describe how to best test these changes. Please provide a [minimal, complete and verifiable example](https://stackoverflow.com/help/mcve) if possible, you can use the follow template as a start:

```cpp
#include <SFML/Graphics.hpp>
#include <iostream>

int main()
{
    sf::VideoMode vm = sf::VideoMode({1280, 720});
    sf::Vector2u aspectRatio = vm.getAspectRatio();

    std::cout << aspectRatio.x << " " << aspectRatio.y << std::endl;
}
```

### CODE REVIEW

**Comment by ChrisThrasher** [comment]
What's the intended use case for this feature?

**Comment by dustdfg** [comment]
> What's the intended use case for this feature?

Make easier to work with aspect ratio of monitors and sometimes arbitary windows

**Comment by ChrisThrasher** [comment]
What specifically about working with aspect ratios?

File: src/SFML/Window/VideoMode.cpp [hunk 1]
```diff
@@ -85,6 +87,14 @@ bool VideoMode::isValid() const
     return std::find(modes.begin(), modes.end(), *this) != modes.end();
 }
 
+////////////////////////////////////////////////////////////
+Vector2u VideoMode::getAspectRatio() const;
```
Comment (Bromeon) [inline src/SFML/Window/VideoMode.cpp:91]: Semicolon.

File: src/SFML/Window/VideoMode.cpp [hunk 2]
```diff
@@ -27,8 +27,10 @@
 ////////////////////////////////////////////////////////////
 #include <SFML/Window/VideoMode.hpp>
 #include <SFML/Window/VideoModeImpl.hpp>
+#include <SFML/System/Vector2.hpp>
```
Comment (Bromeon) [inline src/SFML/Window/VideoMode.cpp:30]: This #include is not needed again.

**Review by Bromeon — COMMENTED** [review]
This seems like a useful feature to me! Would be good to have 5-10 tests though.

**Comment by eXpl0it3r** [comment]
We're not accepting any new features for SFML 2.6, please retarget it to the master branch

**Comment by Bromeon** [comment]
Superseded by #2414.

**Pull request closed without merging**